src/test/java/org/opentripplanner/generate/doc/framework/DocBuilder.java

```diff
@@ -13,6 +19,8 @@
 @SuppressWarnings("UnusedReturnValue")
 public class DocBuilder {
 
+  public static final ObjectMapper objectMapper = new ObjectMapper();
+  public static final DefaultPrettyPrinter prettyPrinter = new DefaultPrettyPrinter();
```

Comment:
If static the names need to be upper case, but I do not think they should be static. 

Also the `prettyPrintJson` configures these instances. Two option, configure these during class initialization or make the ObjectMapper method local.

Reply:
Done in 56edbb3c1429240336ebedac6f43ade74d850ae1.